insights/formats/text.py

```diff
@@ -2,7 +2,11 @@
 from pprint import pprint
 from insights import dr, datasource, rule
 from insights.core.context import ExecutionContext
-from insights.formats import Formatter
+from insights.formats import Formatter, render
+from colorama import Fore, Style
+
+
+_counts = {'skip': 0, 'pass': 0, 'rule': 0, 'metadata': 0, 'metadata_key': 0}
```

Comment:
I'd put this in the class `__init__` and just call it `counts`.

Reply:
Yep, that makes sense. Then I guess other formatters could use it if they wanted to.